sdk/python/requirements.in

```diff
@@ -10,9 +10,8 @@ docstring-parser>=0.7.3,<1
 # https://github.com/googleapis/python-api-core/releases/tag/v1.31.5
 google-api-core>=1.31.5,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0
 google-auth>=1.6.1,<3
-# `Blob.from_string` was introduced in google-cloud-storage 1.20.0
-# https://github.com/googleapis/python-storage/blob/master/CHANGELOG.md#1200
-google-cloud-storage>=1.20.0,<2
+# https://github.com/googleapis/python-storage/blob/main/CHANGELOG.md#221-2022-03-15
+google-cloud-storage>=2.2.1,<3
```

Comment:
Can you please add a release note for this change following the same format as https://github.com/kubeflow/pipelines/blob/2509f115792dca813926a8cf46762af4ded57dcd/sdk/RELEASE.md?plain=1#L21

Reply:
IIRC, we only use this library in v1 code (under `deprecated` folder in `master` branch). So we should be able to get rid of this dependency in the near future. 
Meanwhile, this would be long-lived in `sdk/release-1.8` branch. So can you please also send a PR to change it there? Thanks!

Reply:
The PR to `sdk 1.8` https://github.com/kubeflow/pipelines/pull/7502/files

Reply:
> Can you please add a release note for this change following the same format as
> 
> https://github.com/kubeflow/pipelines/blob/2509f115792dca813926a8cf46762af4ded57dcd/sdk/RELEASE.md?plain=1#L21

added